modules/ROOT/pages/salesforce/salesforce-connector-100.adoc

```diff
@@ -0,0 +1,204 @@
+= Salesforce Connector 10.0 - Get Started - Mule 4
```

Comment:
REEDIT BEGINS HERE

Instructions for edit:

This one is in good shape. It looks like you have previously edited it, I made very few changes to it.